src/pages/changelog/2025-01-15.mdx

```diff
@@ -0,0 +1,31 @@
+---
+date: 2025-01-25
+title: Signatory UI settings and webhook signing
+---
+import Layout from '../../layouts/mdx';
+
+export default Layout;
+
+## Signatory UI settings
+
+Historically signature UI settings for users have been configured globally on the signature order.
+
+This had some drawbacks as each users may be in different countries and have different ideal languages.
```

Comment:
Suggestion: This had some drawbacks, as users might reside in different countries and prefer different languages.